modin/engines/base/io/text/csv_reader.py

```diff
@@ -110,24 +110,24 @@ def _read(cls, filepath_or_buffer, **kwargs):
                     skiprows += header + 1
                 elif hasattr(header, "__iter__") and not isinstance(header, str):
                     skiprows += max(header) + 1
-                for _ in range(skiprows):
-                    f.readline()
+                cls.offset(
+                    f,
+                    nrows=skiprows,
```

Comment:
Why not `skiprows=skiprows`?

Reply:
`skiprows=skiprows` means - read the whole file, but skip first `skiprows` rows, but what we want to do here is just "read" first `skiprows` rows